kolibri/plugins/learn/assets/src/vue/toolbar/index.vue

```diff
@@ -10,9 +10,8 @@
         id="chan-select"
         name="chan-select"
         v-model="getCurrentChannel"
```

Comment:
While you are also in here, maybe also replace v-model by the correct implementation you had mentioned in https://github.com/learningequality/kolibri/pull/358? :sweat_smile: